docs/src/modules/components/Demo.js

```diff
@@ -332,6 +333,9 @@ const DemoCodeViewer = styled(HighlightedCode)(() => ({
     maxWidth: 'initial',
     borderRadius: 0,
   },
+  '& .MuiCode-copy': {
+    visibility: 'hidden',
```

Comment:
Similar but closer to the intention
```suggestion
    display: 'none',
```